LICENSES/fetch-licenses.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -u
+#
+# Requires license-detector from https://github.com/go-enry/go-license-detector/releases/tag/v4.3.0 
+# to be installed in the current directory.
+
+function cleanup {
+  rm  -rf tmp
+}
+trap cleanup EXIT
+
+check_manually=""
+exceptions="github.com/opencontainers/go-digest" # Apache2.0 but reported as CC-BY-SA-4.0
+
+dirs=$(grep -E '^[[:blank:]]+.+\..+/.+' ../go.mod | awk '{print $1}')
+
+for dir_versioned in $dirs; do
+  found="no"
+  for except in $exceptions; do
+    if [ $except == "$dir_versioned" ]; then found="yes"; break; fi
+  done
+  if [ "$found" == "yes" ]; then continue; fi
+
+  echo $dir_versioned":"
+  dir=$dir_versioned
+
+  # versioned paths can not be used by the license-detector
+  if [[ $dir =~ /v[0-9]$ ]]; then
+    dir=${dir%/*}
+  fi
+
+  if [ -n "${dir%%github.com/*}" ]; then
+    repo=$(curl -Ls "https://$dir?go-get=1" | xml2 2>/dev/null | grep -E '/meta/@content=.+\..+ git ' | awk '{print $3}')
+  else
+    # e.g. github.com/moby/sys/user -> github.com/moby/sys
+    repo_dir=$(echo $dir|cut -d'/' -f1-3)
+    repo="https://$repo_dir"
+  fi
+  echo "  $repo"
+
+  if [ -z "${repo%%https://github.com/*}" ]; then
+    json=$(curl -sL "https://api.github.com/repos/$(echo $repo | cut -d'/' -f4-)" -H 'Accept: application/vnd.github.preview')
+    stars=$(echo $json | jq -r '.stargazers_count')
+    forks=$(echo $json | jq -r '.forks')
+    license=$(echo $json | jq -r '.license.spdx_id')
+    created=$(echo $json | jq -r '.created_at')
```

Comment:
No need to fork all these processes here:
```suggestion
    read -r stars forks license created < \
         <(jq -r '"\(.stargazers_count) \(.forks) \(.license.spdx_id) \(.created_at)"' <<< "$json")
```